README.adoc

```diff
@@ -64,15 +64,31 @@ server, then any logging software would do the trick.
 [role='command']
 include::{common-includes}/gitclone.adoc[]
 
-
-For this guide, use Zipkin as your distributed tracing system. You can find the installation instructions
+For this guide, Zipkin is used as the distributed tracing system. You can find the installation instructions
 for Zipkin at the Zipkin https://zipkin.io/pages/quickstart.html[quickstart page^]. You are not required
 to use Zipkin, but keep in mind that you might need more instructions that are not listed here if you choose
 to use another tracing system.
 
+// static guide instructions:
+ifndef::cloud-hosted[]
 Before you proceed, make sure that your Zipkin server is up and running. By default, Zipkin can be found
 at the {zipkin-url}[{zipkin-url}^] URL.
+endif::[]
+
+// cloud-hosted guide instructions:
+ifdef::cloud-hosted[]
+Start Zipkin by running the following command:
+```
+docker run -d --name zipkin -p 9411:9411 openzipkin/zipkin
+```
+{: codeblock}
 
+Before you proceed, make sure that your Zipkin server is up and running.
+Select **Launch Application** from the menu of the IDE, 
```

Comment:
Select **Launch Application** from the menu of the IDE, 
type in **9411** to specify the port number for the Zipkin service, and click the **OK** button.
--->
Select **Launch Application** from the menu of the IDE and 
type **9411** to specify the port number for the Zipkin service. Click the **OK** button.